UPGRADE-6.7.md

```diff
@@ -2207,3 +2207,9 @@ The `technical_name` column will be made non-nullable for the `payment_method` a
 Plugin developers will be required to supply a `technicalName` for their payment and shipping methods.
 
 Merchants must review their custom created payment and shipping methods for the new `technicalName` property and update their methods through the administration accordingly.
+
+## Cache ID loaded by Database is removed
```

Comment:
don't we do that via a changelog file and the "next major" thingy section? :thinking: